egs/tedlium/s5_r2/local/Libri_data_prep.sh

```diff
@@ -0,0 +1 @@
+/home/moses/kaldi/egs/librispeech/s5/local/data_prep.sh
```

Comment:
Actually, before I merge you should probably fix this and similar to
(a)  be a relative pathname (this won't work on other machines), and
(b) not have an uppercase letter in it.  better to call  it librispeech_data_prep.sh, for consistency.